requirements-dev.txt

```diff
@@ -1,13 +1,11 @@
--c requirements.txt
--r requirements.txt
```

Comment:
These are redundant because `-e .` will already include them since they are dependencies of nbautoexport.